README.md

```diff
@@ -42,6 +42,25 @@ Crystalball.foresee do |predictor|
 end
 ```
 
+## Map Generator
+
+There are different map generator strategies that can (and should) be used together for better predictions. Each one has its own benefits and drawbacks, so they should be configured to best fit your needs.
+
+- **CoverageStrategy** uses coverage information to detect which files are covered by the given spec (i.e. the files that, if changed, may potentially break the spec);
+- **AllocatedObjectsStrategy** looks for the files in which the objects created during the spec execution are defined. It is considerably slower than `CoverageStrategy`.
+
+## Predictor
+
+The predictor can also be customized with different strategies:
+
+- **AssociatedSpecs** needs to be configured with rules for detecting which specs should be on the prediction.
+  `predictor.use Crystalball::Predictor::AssociatedSpecs.new(from: %r{models/(.*).rb}, to: "./spec/models/%s_spec.rb")`
+  will add `./spec/models/foo_spec.rb` to prediction when `models/foo.rb` changes.
+  This strategy does not depend on a previoulsy generated case map.
+- **ModifiedExecutionPaths** checks the case map and the diff to see which specs are affected by the new or modified files.
+- **ModifiedSpecs**, as the name implies, checks for modified specs. The scope can be modified by passing a regex as argument, which defaults to `%r{\Aspec/.*_spec\.rb\z}`.
```

Comment:
the new default regex is going to be `%r{spec/.*_spec\.rb\z}` 
check https://github.com/toptal/crystalball/pull/51/files